tests/test_tokenization_blenderbot.py

```diff
@@ -16,22 +16,37 @@
 """Tests for Blenderbot Tokenizers, including common tests for BlenderbotSmallTokenizer."""
 import unittest
 
+from transformers import BlenderbotTokenizer, BlenderbotTokenizerFast
 from transformers.file_utils import cached_property
-from transformers.models.blenderbot.tokenization_blenderbot import BlenderbotTokenizer
 
 
 class Blenderbot3BTokenizerTests(unittest.TestCase):
     @cached_property
     def tokenizer_3b(self):
         return BlenderbotTokenizer.from_pretrained("facebook/blenderbot-3B")
 
+    @cached_property
+    def rust_tokenizer_3b(self):
```

Comment:
(nit) maybe call this `fast_tokenizer` for consistancy

Reply:
In test files, it seems to me the `rust_tokenizer` naming is used instead of `fast_tokenizer`. However, I can change it here :)